site/docs/integrations/docker-extension.md

```diff
@@ -2,4 +2,115 @@
+
+### Steps
+
+1. **Installing the extension**
+
+   Navigate to this [link](https://open.docker.com/extensions/marketplace?extensionId=livecycle/docker-extension) or search for "Livecycle" in the Docker Desktop Extensions Marketplace. Click the "Install" button to install the extension.
+
+   ![Livecycle Docker Extension](docker-ext-1.png)
+
+2. **Setting up a Livecycle account**
+
+   Once you have installed the extension and opened it, you will be greeted with a login screen. You can choose to log in with your GitHub account or Google account. If you previously used Livecycle and created an organization, you can log in with your Livecycle account.
+
+   ![Livecycle Docker Extension](docker-ext-2.png)
+
+3. **Getting shareable URLs**
+   
+   As soon as you log in, you will be able to see a list of running docker compose applications and all the services that are running in them. To get a public shareable URL for every service, turn on the toggle next to the compose application name. After that, you will be prompted to choose the access level. 
+
+   ![Livecycle Docker Extension](docker-ext-3.png)
+
+   You can choose between public and private access. If you choose public access, you will get a public URL that you can share with anyone. If you choose private access, you will get a private URL that you can share with your team members. Then click on the "Share" button to get the shareable URL.
+
+   ![Livecycle Docker Extension](docker-ext-4.png)
+
+4. **Accessing the shared URL**
+
+   Once you have shared the URL with your team members, they can access the URL in their browser. They will be able to see the application running in their browser.
+
+   ![Livecycle Docker Extension](docker-ext-5.png)
+
+5. **Accessing Livecycle dashboard**
+
+   You can also access the Livecycle dashboard to see the logs and debug your application. Click on the "Open Link" button to open the Livecycle dashboard. On the dashboard, you can see all the running applications and services. You can also see the public and private URLs for each service.
+
+   ![Livecycle Docker Extension](docker-ext-6.png)
+
+6. **Debugging, inspecting, and logging**
+
+   Once you have opened the Livecycle dashboard, you can see all the environments/apps that are running. Click on the name of the environment for which you want to see the logs, terminal, etc. You can view the logs, terminal, and container inspection for each service.
+
+   ![Livecycle Docker Extension](docker-ext-7.png)
+
+That's it! You have successfully installed the Livecycle Docker Extension and shared your local development environment with your team.
+
+## FAQ
+
+<details>
+  <summary>How Livecycle Docker Extension is different from ngrok?</summary>
+  <p>
+    The main difference is the collaboration aspect. Livecycle allows you to share your local development environment with your team, so you can get feedback much earlier in the development workflow.
```

Comment:
I'm not sure that is the difference, as ngrok allows sharing your local dev env as well.

I think the difference is that our Extension is a gateway to demonstrate the collaboration experience you can get using Preevy on remote environments. if you find the collaboration flow useful, you can expand it to create preview environments using the same tool.